client/features/support/navigation.js

```diff
@@ -0,0 +1,27 @@
+const scope = require('./scope');
+
+const goto = async (page = '') => {
+  if (!scope.browser) {
+    scope.browser = await scope.driver.launch({
+      ignoreHTTPSErrors: true,
+      args: ['--no-sandbox', '--headless'],
```

Comment:
may be those args could be related to env var. it could be great to watch the process with browser opened to debug
Something like this:
```
    const isCIEnv = !!process.env.CI;
    
    if (!scope.browser) {
      scope.browser = await scope.driver.launch({
        ignoreHTTPSErrors: true,
        args: ['--no-sandbox', '--disable-setuid-sandbox'],
        headless: isCIEnv,
        slowMo: isCIEnv ? 0 : 2,
      });
    }
```